command/resource/read/read_test.go

```diff
@@ -32,23 +36,13 @@ func TestResourceReadInvalidArgs(t *testing.T) {
 		"missing file path": {
 			args:           []string{"-f"},
 			expectedCode:   1,
-			expectedErrMsg: "Please input file path",
+			expectedErrMsg: "Failed to parse args: flag needs an argument: -f",
 		},
 		"provide type and name": {
 			args:           []string{"a.b.c"},
 			expectedCode:   1,
 			expectedErrMsg: "Must specify two arguments: resource type and resource name",
 		},
-		"provide type and name with -f": {
-			args:           []string{"a.b.c", "name", "-f", "test.hcl"},
-			expectedCode:   1,
-			expectedErrMsg: "You need to provide all information in the HCL file if provide its file path",
-		},
-		"provide type and name with -f and other flags": {
-			args:           []string{"a.b.c", "name", "-f", "test.hcl", "-namespace", "default"},
-			expectedCode:   1,
-			expectedErrMsg: "You need to provide all information in the HCL file if provide its file path",
-		},
```

Comment:
Why did we remove these?
Also, for these cases we should just be displaying a warning message that the `-f` flag will be ignored.

Reply:
yes, restored the test and display the warning message that the `-f` will be ignored.